### PR TITLE
Updated report styles

### PR DESCRIPTION
**Pull request by chowse**
Updated the Lighthouse report's styling to better match the DevTools UI (with options to restyle it using CSS variables).

### CODE REVIEW

**Comment by paulirish** [comment]
Before and after

image

**Comment by chowse** [comment]
I'm using a 12px baseline font for this design. For the standalone report, I'd prefer to use a 14px baseline (should only require tweaking a few CSS variables). Just let me know how we'll differentiate a DevTools- and CLI-generated report and I can make the adjustments.

**Comment by paulirish** [comment]
>  Just let me know how we'll differentiate a DevTools- and CLI-generated report 

@chowse we have `.lh-devtools` on the lh-root container in devtools.  

Also the `.lh-narrow` i mentioned before? It's only applied in standalone report.

**Comment by patrickhulce** [comment]
> Also the .lh-narrow i mentioned before? It's only applied in standalone report. So probably not as useful as I indicated before. :)

Adding it to devtools is just a few line change though if we need it

**Comment by paulirish** [comment]
And fwiw here's this PR applied within devtools:

image

**Comment by chowse** [comment]
The same report without the dense DevTools styling:

image

File: lighthouse-core/report/v2/report-styles.css [hunk 1]
```diff
@@ -520,25 +611,33 @@ span.lh-node:hover {
...
 
 .lh-audit-group {
-  padding-top: var(--default-padding);
-  padding-bottom: var(--default-padding);
+  padding-top: var(--lh-audit-group-vpadding);
   border-top: 1px solid var(--report-secondary-border-color);
+  padding-left: var(--expandable-indent);
+}
+
+.lh-audit-group[open] {
```
Comment (patrickhulce) [inline lighthouse-core/report/v2/report-styles.css]: does this have any content?

File: lighthouse-core/report/v2/report-styles.css [hunk 2]
```diff
@@ -639,24 +750,20 @@ span.lh-node:hover {
 }
 
 .lh-category .lh-audit {
-  margin-left: calc(var(--default-padding) * 2);
```
Comment (patrickhulce) [inline lighthouse-core/report/v2/report-styles.css]: are none of these rules needed anymore then?

File: lighthouse-core/report/v2/templates.html [hunk 3]
```diff
@@ -347,11 +369,11 @@ <h1 class="leftnav__header__title">Lighthouse</h1>
...
     .lh-gauge {
-      --circle-size: 80px;
+      --circle-size: calc(2.5 * var(--header-font-size));
       --circle-size-half: calc(var(--circle-size) / 2);
-      --circle-background: #eee;
-      --circle-border-width: 8px;
-      --inset-size: calc(var(--circle-size) - var(--circle-border-width));
+      --circle-background: #ccc;
+      --circle-border-width: 2px;
+      --inset-size: calc(var(--circle-size) - 2* var(--circle-border-width));
```
Comment (patrickhulce) [inline lighthouse-core/report/v2/templates.html]: nit: `2 *`

**Comment by paulirish** [comment]
one thing I lose from the previous design is the green encoding of the passing group


image


in this view i'd love to more clearly see that we do have passing items in the group there.. maybe two overlapping green checks on the right of "8 Passed Audits" ?

**Comment by ev1stensberg** [comment]
I can submit a fix to #2385 once this gets merged

**Comment by brendankenny** [comment]
@paulirish do you have a sense of remaining things to resolve? Can we make a list?

**Comment by paulirish** [comment]
Yah. I have a branch pushed with some more work. I'll prep a list of remaining work tonight.

**Pull request merged**